utilities/src/main/java/org/craftercms/commons/crypto/impl/PbkAesTextEncryptor.java

```diff
@@ -50,7 +57,21 @@ private static Key generateKey(String password, String salt) throws CryptoExcept
     }
 
     public PbkAesTextEncryptor(String password, String salt) throws CryptoException {
-        super(generateKey(password, salt));
+        super(generateKey(password, Base64.decodeBase64(salt)));
```

Comment:
I'm not sure this is going to work. We don't want to decode the salt until we do the encryption/decryption. I want to be able to pass a salt in new installations that is not a Base 64 string